### PR TITLE
feat(std): add new go 1.16 std packages

### LINKED ISSUE #43: Go 1.16 support
Since updating to Go 1.16 a number of stdlib packages appear to be sorted in the wrong groups.

The following expected example:

```go
import (
	"embed"
	"io/fs"
	"io/ioutil"
	"net"
	"net/http"
	"os"
	"runtime"
	"strconv"
	"strings"

	duoapi "github.com/duosecurity/duo_api_golang"
	"github.com/fasthttp/router"
	"github.com/valyala/fasthttp"
	"github.com/valyala/fasthttp/expvarhandler"
	"github.com/valyala/fasthttp/fasthttpadaptor"
	"github.com/valyala/fasthttp/pprofhandler"

	"github.com/authelia/authelia/internal/configuration/schema"
	"github.com/authelia/authelia/internal/duo"
	"github.com/authelia/authelia/internal/handlers"
	"github.com/authelia/authelia/internal/logging"
	"github.com/authelia/authelia/internal/middlewares"
)
```

Is sorted into the following incorrect example:

```go
import (
	"io/ioutil"
	"net"
	"net/http"
	"os"
	"runtime"
	"strconv"
	"strings"

	"embed"
	duoapi "github.com/duosecurity/duo_api_golang"
	"github.com/fasthttp/router"
	"github.com/valyala/fasthttp"
	"github.com/valyala/fasthttp/expvarhandler"
	"github.com/valyala/fasthttp/fasthttpadaptor"
	"github.com/valyala/fasthttp/pprofhandler"
	"io/fs"

	"github.com/authelia/authelia/internal/configuration/schema"
	"github.com/authelia/authelia/internal/duo"
	"github.com/authelia/authelia/internal/handlers"
	"github.com/authelia/authelia/internal/logging"
	"github.com/authelia/authelia/internal/middlewares"
)
```

### PR DESCRIPTION
**Pull request by james-d-elliott**
This adds the new std packages listed on https://golang.org/doc/go1.16 to the StdPackages map.

Fixes #43

### CODE REVIEW

File: pkg/std/package_list.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ var StdPackages = map[string]struct{}{
 	"debug/macho":                          {},
 	"debug/pe":                             {},
 	"debug/plan9obj":                       {},
+	"embed":                                {},
```
Comment (incu6us) [inline pkg/std/package_list.go:50]: this file is autogenerated and after regeneration all changed stuff will be missed

**Comment by incu6us** [comment]
also, not all packages added for go 1.16

**Comment by incu6us** [comment]
actually, I've already done with the fix and will close your PR.

anyway, thanks for your attention to the issues and your contribution.

**Comment by james-d-elliott** [comment]
> 
> 
> actually, I've already done with the fix and will close your PR.
> 
> anyway, thanks for your attention to the issues and your contribution.

No worries! Glad you have a way to auto-generate it. Half of the ones I saw in your commit were not in mine or in release notes.

**Pull request closed without merging**